pkg/controllers/vrep/suite_test.go

```diff
@@ -97,3 +104,92 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+const testAccessKey = "dummy"
+const testSecretKey = "dummy"
+const testTargetVdbUID = "jklmno-pqr"
+
+func createS3CredSecret(ctx context.Context, vdb *v1.VerticaDB) {
```

Comment:
Do we need createS3CredSecret or deleteCommunalCredSecret? we don't need communal storage in this API

Reply:
will remove them